### PR TITLE
Adds a squeak emote

### PR DESCRIPTION
**Pull request by CydiaButt13**
Adds a cute squeak for insects.

[Changelogs]: Adds the msqueak emote, a little insect squeak noise.

:cl:
add: added msqueak emote
soundadd: modular_citadel/sound/voice/mothsqueak.ogg
:cl:

[why]:
I think the emote would be nice, because the emote's cute and I think that insects should get their little emote too.

### CODE REVIEW

**Review by silicons — DISMISSED** [review]

**Review by silicons — CHANGES_REQUESTED** [review]
wait no no hardcoded species/mobtype checks please, make it an allowed typecache.

**Comment by CydiaButt13** [comment]
Okay, there's a typecache now.

File: modular_citadel/code/modules/mob/cit_emotes.dm [hunk 1]
```diff
@@ -198,15 +198,16 @@
...
-		user.nextsoundemote = world.time + 7
-		playsound(user, 'modular_citadel/sound/voice/mothsqueak.ogg', 50, 1, -1)
-	. = ..()
\ No newline at end of file
+    if(ishuman(user))
+        if(user.nextsoundemote >= world.time)
+            return
+        user.nextsoundemote = world.time + 7
+        playsound(user, 'modular_citadel/sound/voice/mothsqueak.ogg', 50, 1, -1)
+. = ..()
```
Comment (king5327) [inline modular_citadel/code/modules/mob/cit_emotes.dm]: This last line is not tabbed up. Needs a tab. or it'll compile not well.

**Comment by Drewfuscat** [comment]
Perhaps instead of message "emits a tiny squeak!", message "lets out a tiny squeak!" similar to other emotes

File: modular_citadel/code/modules/mob/cit_emotes.dm (hunk 1 above)
Comment (CydiaButt13) [reply modular_citadel/code/modules/mob/cit_emotes.dm]: Should be fixed now

**Comment by king5327** [comment]
> 
> 
> Perhaps instead of message "emits a tiny squeak!", message "lets out a tiny squeak!" similar to other emotes

Lets out is like the laziest writing, though. Unless there's an actual physical object being released, chances are good there's a verb that could replace "lets out" and sound better.

**Review by deathride58 — DISMISSED** [review]

File: modular_citadel/code/modules/mob/cit_emotes.dm [hunk 2]
```diff
@@ -194,4 +194,20 @@
...
\ No newline at end of file
+	. = ..()
+
+
+/datum/emote/living/mothsqueak
+	key = "msqueak"
+	key_third_person = "lets out a tiny squeak"
+	message = "lets out a tiny squeak!"
+	emote_type = EMOTE_AUDIBLE
+	mob_type_allowed_typecache = list(/mob/living/carbon/)
```
Comment (silicons) [inline modular_citadel/code/modules/mob/cit_emotes.dm]: no / on the end

File: modular_citadel/code/modules/mob/cit_emotes.dm [hunk 3]
```diff
@@ -194,4 +194,20 @@
...
+
+/datum/emote/living/mothsqueak/run_emote(mob/living/user, params)
+	if(ishuman(user))
```
Comment (silicons) [inline modular_citadel/code/modules/mob/cit_emotes.dm]: remove the ishuman if there's already a typecache

**Review by silicons — CHANGES_REQUESTED** [review]

File: modular_citadel/code/modules/mob/cit_emotes.dm [hunk 4]
```diff
@@ -194,4 +194,19 @@
...
+
+/datum/emote/living/mothsqueak
+	key = "msqueak"
+	key_third_person = "lets out a tiny squeak"
+	message = "lets out a tiny squeak!"
+	emote_type = EMOTE_AUDIBLE
+	mob_type_allowed_typecache = list(/mob/living/carbon)
+
+/datum/emote/living/mothsqueak/run_emote(mob/living/user, params)
+		if(user.nextsoundemote >= world.time)
```
Comment (silicons) [inline modular_citadel/code/modules/mob/cit_emotes.dm]: this entire block needs to be shifted down in terms of indentation so this compiles

**Review by silicons — APPROVED** [review]

**Pull request merged**